pilot/pkg/bootstrap/mesh.go

```diff
@@ -51,16 +60,18 @@ func (s *Server) initMeshConfiguration(args *PilotArgs, fileWatcher filewatcher.
 	if _, err = os.Stat(args.MeshConfigFile); !os.IsNotExist(err) {
 		s.environment.Watcher, err = mesh.NewFileWatcher(fileWatcher, args.MeshConfigFile)
 		if err == nil {
+			// Normal install no longer uses this mode - testing and special installs still use this.
```

Comment:
This seems to contradict the comment, or I misunderstand one. The comment seems to say we will merge file+configmap, but here it looks like we only read the file?

Reply:
Yes, if file is present - it'll only read the file, as is the current behavior.